guides/user/kubernetes-v2/patch-manifest/index.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: single
+title:  "Patch Kubernetes Manifests"
+sidebar:
+  nav: guides
+---
+
+{% include alpha version="1.8" %}
+
+{% include toc %}
+
+This guide shows the basics of how to update a Kubernetes resource in place using the patch manifest stage for the [Kubernetes Provider V2](/setup/install/providers/kubernetes-v2) provider.
+
+There are a few steps:
+
+* [Specify which manifest to patch](#specify-the-resource-to-patch)
+
+* [Specify the patch content](#specify-your-patch-content)
+
+* [(Optional) Override artifacts in the patch content](#override-artifacts)
+
+  Optionally, you can override some artifacts (as fields) at run time (for
+    example, which Docker image to use.)
+
+* [(Optional) Override patch specific options](#specify-patch-options)
+
+## Specify the resource to patch
+
+Start by selecting the __Patch (Manifest)__ stage from the stage selector:
```

Comment:
I'd number these.

1. Select the __Patch (Manifest)__ stage from the stage selector: